README.md

```diff
@@ -0,0 +1,227 @@
+![Release](https://img.shields.io/pypi/v/auth0-python)
+[![Codecov](https://img.shields.io/codecov/c/github/auth0/auth0-python)](https://codecov.io/gh/auth0/auth0-python)
+![Downloads](https://img.shields.io/pypi/dw/auth0-python)
+[![License](https://img.shields.io/:license-MIT-blue.svg?style=flat)](https://opensource.org/licenses/MIT)
+[![CircleCI](https://img.shields.io/circleci/build/github/auth0/auth0-python)](https://circleci.com/gh/auth0/auth0-python)
+
+<div>
+📚 <a href="#documentation">Documentation</a> - 🚀 <a href="#getting-started">Getting started</a> - 💻 <a href="#api-reference">API reference</a> - 💬 <a href="#feedback">Feedback</a>
+</div>
```

Comment:
Needed to use HTML here, as well as actual emoji's rather than github's `:book:`, to ensure it renders as expected on our generated documentation.